docs/user-guide/kubectl-overview.md

```diff
@@ -135,6 +135,66 @@ For explanation, here I gave some mostly often used `kubectl` command examples.
 		// Starts streaming of logs from pod <pod-name>, it is something like 'tail -f'.
 		$ kubectl logs -f <pod-name>
 
+                // Get the logs from a previous run of a particular pod
+                $ kubectl logs --previous <pod-name>
+
+## Kubectl output
+
+`kubectl` supports a variety of output formats for different use cases.  The output format is selected with the `-o/--output` flag.
+
+The following options are supported:
+
+Output Format | Description
+--------------| -----------
+<default>     | Human readable format
+`-o=wide`     | Human readable format but with more columns
+`-o=yaml`     | YAML formatted API object
+`-o=json`     | JSON formatted API object
+`-o=name`     | Print the resource name and nothing else
+`-o=jsonpath=<template>` | Print fields selected by a [jsonpath](jsonpath.md) expression.
+`-o=jsonpath-file=<filename>` | Print the fields specified by [jsonpath](jsonpath.md) expressions in <filename>.
+`-o=custom-columns=<spec>` | Print a table using a comma separated list of [custom columns](#custom-columns)
+`-o=custom-columns-file=<filename>` | Print a table using a [custom columns](#custom-columns) template.
+
+### Sorting list objects
+
+Users can sort list objects (e.g. `PodList`) by any numeric or string field using the `--sort-by` flag.  The field is selected using a
+[jsonpath](jsonpath.md) expression.
+
+For example:
+
+```sh
+kubectl get pods --sort-by=.metadata.name
+```
+
+Prints out a list of Pods, sorted by their name
+
+### Custom columns
+
+The `custom-columns` printer enables users to output user-specified columns of object data.  There are two ways to specify these custom columns.
```

Comment:
Please revise this whole section to:


To define custom columns and output only the details that you want into a table, you can use the `custom-columns` option. You can choose to define the custom columns inline or use a template file:

 * Inline:

      ```console
      $ kubectl get pods <pod-name> -o=custom-columns=NAME:.metadata.name,RSRC:.metadata.resourceVersion
     ```

 * Template file:

     ```console
     $ kubectl get pods <pod-name> -o=custom-columns-file=template.txt
     ```
     where the `template.txt` file contains:  
     ```
      NAME                    RSRC
      metadata.name           metadata.resourceVersion
     ```

The result of running either command is:

```console
NAME           RSRC
submit-queue   610995
```